aws/terraform/metaflow/outputs.tf

```diff
@@ -43,9 +43,9 @@ output "METAFLOW_SFN_DYNAMO_DB_TABLE" {
   description = "AWS DynamoDB table name for tracking AWS Step Functions execution metadata."
 }
 
-output "metaflow_profile_json" {
-  value       = module.metaflow.metaflow_profile_json
-  description = "Metaflow profile JSON object that can be used to communicate with this Metaflow Stack. Store this in `~/.metaflow/config_[stack-name]` and select with `$ export METAFLOW_PROFILE=[stack-name]`."
+output "metaflow_profile_configuration" {
+  value       = "metaflow configure import ${path.module}/${local.metaflow_config_filename}"
+  description = "Output the command to import Metaflow configuration"
```

Comment:
Oh, I see. So the output is the import command with the json file path already filled in.

Does "Run this output in a shell to import the Metaflow configuration" make sense?